packages/framework/collabspaces/src/test/collabSpaces.spec.ts

```diff
@@ -34,28 +33,26 @@ import { createCollabSpace } from "../factory";
 
 import { CounterFactory, ISharedCounter } from "./counterFactory";
 
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
remove

Reply:
It's not hard to remove, as we have things like loader, summarizer that needs to be cleared (assigned undefined) in afterEach(), so their type needs to include undefined. But I do not want to test every single place such things are used for undefined. If it happens to be undefined, it would crash either way.